openquakeplatform/openquakeplatform/static/js/irv_viewer_d3_tree.js

```diff
@@ -451,15 +451,50 @@
                     return "-1em";
                 })
                 .attr("text-anchor", function(d) { return "end"; })
+                // Convert long attribute names text into acronyms
                 .text(function(d) {
-                    if (d.isInverted) {
+                    if (d.isInverted && d.name.length <= 20) {
                         return "- " + d.name;
-                    } else {
+                    } else if (d.isInverted && d.name.length > 20) {
+                        var matches = d.name.match(/\b(\w)/g);
+                        var acronym = matches.join('').toUpperCase();
+                        return "- " + acronym;
+                    } else if (d.name.length <= 20) {
                         return d.name;
+                    } else if (d.name.length > 20) {
+                        var matches = d.name.match(/\b(\w)/g);
+                        var acronym = matches.join('').toUpperCase();
+                        return acronym;
```

Comment:
It would be good to avoid duplicating the construction of `matches` and `acronym`. You could re-arrange conditions differently. So, if name length exceeds 20 characters, the acronym is built, and it is returned with a - if d.isInverted, or without the minus otherwise.